.eslintrc.json

```diff
@@ -18,9 +18,6 @@
     "@typescript-eslint/triple-slash-reference": "warn",
     "no-async-promise-executor": "warn",
     "no-control-regex": "warn",
-    "no-debugger": "warn",
-    "no-empty": "warn",
-    "no-extra-boolean-cast": "warn",
     "no-extra-semi": "warn",
```

Comment:
@pleerock I haven't thought about this until now, but after taking a deeper look into the code, I would strongly recommend using semicolons for delimiting statements (here and in the prettier config).

I know this is rather formatting, and it would create a lot of conflicts for existing PRs, and maybe it should be done in a separate PR possibly at a moment when there aren't too many pending PRs, but it omitting them can cause a lot of unexpected behaviors.

If you search for ` ;` (space followed by semicolon) in the entire project (ignore the few places where eslint says it's unnecessary), all those are occurrences are situations where something undesired could have happened if the semicolon was omitted, e.g. after some refactor.

Reply:
I prefer formatting without semicolons. Yeah there are some cases where it can only work with `;(...)` statements, but that's fine.